samples/drivers/jesd216/boards/nrf52840dk_nrf52840_spi.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_SPI=y
+CONFIG_SPI_NOR=y
+CONFIG_NORDIC_QSPI_NOR=n
```

Comment:
This seems unneeded, too. It can be `y` only when a "nordic,qspi-nor" compatible node is enabled, and the default `mx25r64` is deleted in the `.overlay` file.